tools/arxiv/scripts/check_existing_papers.py

```
@@ -0,0 +1,195 @@
+                        existing_in_db.add(paper_id)
+                    elif status == 'FAILED':
+                        failed_in_db.add(paper_id)
+        
+        # Calculate statistics
+        total_papers = len(paper_ids)
+        already_processed = len(existing_in_db)
+        already_failed = len(failed_in_db)
+        not_processed = total_papers - already_processed - already_failed
+        
+        # Print results
+        print(f"📊 Database Status:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)